trestle/core/const.py

```diff
@@ -15,9 +15,20 @@
 # limitations under the License.
 """Core constants module containing all constants."""
 
-DIR_TRESTLE = '_trestle'
-DIR_TRESTLE_BACKUP = f'{DIR_TRESTLE}/backup'
-DIR_TRESTLE_DIST = f'{DIR_TRESTLE}/dist'
+TRESTLE_MODEL_DIRS = [
```

Comment:
In reference to the constant dict I previously mentioned - I think we can exploit these lists much better.

Reply:
Created issue #64.